Create scalb.md (#1238)
Codecademy/docs: content/java/concepts/math-methods/terms/scalb/scalb.md

---

> Title: 'scalb()'

Reviewer: ```suggestion
Title: '.scalb()'
```

Description: 'Returns the first argument multiplied by 2 to the power of the second argument.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Methods'
  - 'Arithmetic'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

The **`Math.scalb()`** method returns the first argument multiplied by 2 to the power of the second argument. 

### Syntax

```pseudo

> Math.scalb(a, b)

Reviewer 2: ```suggestion
Math.scalb(value, scale)
```

```

> The parameter a is of type `double` and the parameter b is of type `int`.

Reviewer 2: ```suggestion
The parameter `value` is of type `double` and the parameter `scale` is of type `int`.
```

## Example

```java

> public class Main {

Reviewer: ```suggestion
// Main.java
public class Main {
```
